src/App.js

```diff
@@ -1,25 +1,104 @@
-import logo from './logo.svg';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
 import './App.css';
+import firebase from './firebase.js';
+// Components 
+import Header from './Components/Header.js';
+import MainContent from './Components/MainContent.js';
+import Footer from './Components/Footer.js';
+import { getDatabase, ref, onValue, push, remove } from 'firebase/database';
+import SavedArt from './Components/SavedArt';
```

Comment:
I like how you have broken up your app into components - very organized and easy to follow!